callbacks.go

```diff
@@ -183,3 +185,17 @@ func (exec *sidecarExecutor) Shutdown(driver executor.ExecutorDriver) {
 func (exec *sidecarExecutor) Error(driver executor.ExecutorDriver, err string) {
 	log.Info("Got error message:", err)
 }
+
+// Check if it should check Sidecar status, assuming enabled by default
+func shouldCheckSidecar(containerConfig *docker.CreateContainerOptions) bool {
+	if _, ok:= containerConfig.Config.Labels["SidecarDiscover"]; !ok {
+		return true
+	}
+
+	value, _ := containerConfig.Config.Labels["SidecarDiscover"]
```

Comment:
Oh I see, you are fetching it twice from the hash. Maybe just slight refactor then